stacksCollectible/index.ts

```diff
@@ -11,7 +11,6 @@ export interface StacksCollectionData {
   collection_id: string | null;
   collection_name: string | null;
   total_nft: number;
-  thumbnail_nfts: NonFungibleToken[]; //stores a max of four nfts
```

Comment:
we're using this prop in mobile. if we're removing it, we should also address the change in mobile. the removal is intentional?

Reply:
true

Reply:
will make a note to change these in mobile then. thanks!

Reply:
removed total_nft too